models/CESM/readme.rst

```diff
@@ -1,15 +1,8 @@
-###########
-CESM README
-###########
-
-Contents
-========
-
-#. `Overview`_
-#. `Terms of Use`_
+CESM
+====
 
 Overview
-========
+--------
```

Comment:
the note about  'doc/setup_guidelines.html'  (10 lines down, but not changed, so I can't comment on it directly) is out-of-date and should be a link to a document.

Reply:
This was rewritten in #195.